components/catalogs/back/src/Application/Service/AttributeValueExtractor/AttributeValueExtractorInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Catalogs\Application\Service\AttributeValueExtractor;
+
+interface AttributeValueExtractorInterface
+{
+    public function extract(
+        array $product,
+        string $attributeCode,
+        string $attributeType,
+        ?string $locale,
+        ?string $scope,
+        ?array $parameters,
+    ): null | string;
+
+    public function support(string $attributeType): bool;
```

Comment:
```suggestion
    public function supports(string $attributeType): bool;
```
to be consistent with what is done in Symfony